helm/publisher/templates/_helpers.tpl

```diff
@@ -0,0 +1,47 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "publisher.name" -}}
+{{- .Chart.Name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
```

Comment:
given the level of reusability of this chart I'm tempted to strip the 63-character checks and if not to support names such as `publisher--my-staging-environment-that-really-really-needs-a-long-name` in favor of `publisher--my-staging-environment-that-really-needs-a-long-name`